### DISCUSSION TITLE
Pin repository

### DISCUSSION THREAD

**Discussion by GuilhermeZorzal** [community/community#124190] [Repositories]
### Select Topic Area

Product Feedback

### Body

On the left side of GitHub, it shows some of the repositories I've worked on. It would be nice to pin repositories inside that list so they appear at the top. Usually, I'm working with several repos at the same time, from different organizations, etc., and every time I open GitHub, I need to search for the repository. Making them pinnable would be awesome, so I can go straight to the repos I'm working on recently.

Making them orderable by recent activity, date of creation, etc., would also be a great alternative to solve the problem.

Maybe there is already something like that, but in case there isn't, I believe it's a good suggestion.

**Comment by davevad93** [comment]
Hi @GuilhermeZorzal ,

Yes, it would be great to add the feature to pin the repos from the dashboard list, the only way to pin your repos is by clicking on `customize your pins` in the `Popular repositories` section of the main profile page, there you can pin up to 6 repos. 
And yes you're right, it would also be very useful to add a feature for sorting them by date, recent activity, stars, forks, programming language etc.
